plugin/src/main/kotlin/com/ritense/plugin/annotation/Plugin.kt

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2015-2022 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ritense.plugin.annotation
+
+@Target(AnnotationTarget.CLASS)
+@Retention(AnnotationRetention.RUNTIME)
+annotation class Plugin(
+    val key: String,
+    val title: String,
+    val description: String,
```

Comment:
Should the title and the description not be arranged on the frontend, because of translations?

Reply:
This can still be done by key. All this does is provide a default.

Reply:
Is the backend responsible for that?

Reply:
In my opinion, if FE does not have any translations available, then it makes sense for BE to provide a default.